<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LocalNet</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Старт!</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox">
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Частота обновления в секундах:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Перезапуск</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
